wiki/Gameplay/Multiplayer/en.md

```diff
@@ -1,434 +1,380 @@
 
-*Only available in osu! mode. **UNRANKED***
+Only players who are alive by the end of the match count towards their teams' totals.
 
-![](img/Multi_tag_co-op.jpg "Tag Coop Interface")
+The panels to the sides the show results for each player in different tabs. The `Total` tab shows the total count of [judgements](/wiki/Gameplay/Judgement) for both teams and their average accuracy.
 
-![](img/Multi_tag_team_vs.jpg "Tag Team Vs Interface")
+#### Tag Coop / Tag Team Vs
 
-Tag Coop makes all players in the Match Setup work together to pass the beatmap, one combo at a time.
-Tag Team Vs is just Tag Coop with red and blue team competition.
+##### Interface
 
-Do note that this team mode is only available in **osu! only** and *unranked* (meaning, it will not be counted to ranked score).
+*Note: This team mode is unranked and only available in osu! mode.*
 
-In this mode, player's slot placement starting from top-to-bottom is important as it will determine when it is the player's turn.
+![](img/Multi_tag_co-op.jpg "Tag Coop interface")
 
-A green arrow will show the player who is currently playing the beatmap's combo.
-The player's name will be shown at the lower right of the screen.
-There will be warning arrows when it is the player's turn and the combo will be in colour until the combo ends.
-The player can adjust for fixed combo colour at Match Setup under **Tag Colour**.
+![](img/Multi_tag_team_vs.jpg "Tag Team Vs interface")
 
-**All players must spin the spinner**.
-Refusal or unable to complete the spinner will break the current combo count.
+`Tag Coop` makes all players in the match work together to pass the beatmap, one combo at a time. `Tag Team Vs` is just `Tag Coop` but with two teams playing against each other.
 
-If a player quits during the match, [Auto](/wiki/Game_modifier/Auto) will take over for the player that left.
+In this mode, the players' slot placement in the lobby determines the order they play in.
 
-For Tag Coop, all players will share the same health bar.
-If the health bar was depleted completely (failed state), the match ends and all players return to the Match Setup immediately without result.
+A green arrow will show the player who is currently playing, and their name will be shown at the lower right. Warning arrows indicate when it is about to be the player's turn and the hit objects will be coloured. The player can set a fixed combo colour at the match setup the [Tag Colour](#tag-colour) section in the match lobby.
 
-For Tag Team Vs, each team has its own respective health bar that are visible to teammates only.
-If the health bar was depleted completely (failed state), the match ends by giving victory to the other team.
+All players must spin the spinner, or the combo will break.
 
-If [Easy (EZ)](/wiki/Game_modifier/Easy) game modifier was used, the two health bar refill provided by the game modifier will still occur but will be depleted completely immediately after *just* to fail the beatmap.
```

Comment:
i kinda have no idea what this was supposed to mean, but i didn't think it was very important so i've removed it for now

Reply:
uhhhhhhh me neither, but I think it means that a player won't have any benefits from the HP bar restoration (= will fail instantly anyway)